### PR TITLE
Release 0.6.20

### PR DESCRIPTION
**Pull request by oooookk7**
AdapterJS: `0.14.1`
socket.io-client: `1.4.8`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,14 +15,14 @@ You'll need a Temasys Account and an App key to use this. [Register here to get
 | -------------- | ---------- | ----------- | --------- | ---------- | ---------- | ---- | ----- |
 | **Platforms:** | Win/Mac/Ubuntu/Android | Win/Mac/Ubuntu/Android | Win/Mac/Ubuntu/Android | Win/Mac | Win | Mac | Win | iOS<br>(`9.x` and below) |
```
Comment (ncurrier) [inline README.md:16]: iOS notes are wrong should be 10.x

File: demo/edge-attachmediastream.js [hunk 2]
```diff
@@ -3,7 +3,7 @@
  * And Edge current does not support re-negotiation for now in the SDK.
  */
 (function () {
-  if (window.webrtcDetectedBrowser === 'edge') {
```
Comment (ncurrier) [inline demo/edge-attachmediastream.js:6]: How was this specific release identified?

**Review by ncurrier — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (oooookk7) [reply README.md:16]: @ncurrier actually this was based off the AdapterJS support for bowser supports which is 9.x and below:
https://github.com/Temasys/AdapterJS#supported-browsers--platform-functionality

Bowser browser doesn't work with 10.x and above.

File: demo/edge-attachmediastream.js (hunk 2 above)
Comment (oooookk7) [reply demo/edge-attachmediastream.js:6]: This was supposed to patch and prevent issues with Edge `15.15019` and above which doesn't seem to need this workaround for rendering audio and video separately. In the later versions of Edge, it appears the video to be black when audio+video is rendered together in the `<video>` element, but `<audio>` and `<video>` elements renders them properly.

After webrtc/adapter resolves the Edge patches for the latest releases, the users can drop-in the new AdapterJS version and use this SDK version to connect with Edge. But if we need to use legacy API for any reasons, we wait will for the proper AdapterJS update and update the SDK again.

Comment (oooookk7) [reply demo/edge-attachmediastream.js:6]: The release was to be able to handle the older Edge and also handle the latest Edge if possible (unless we have to use legacy API depending on time constraints).

File: README.md (hunk 1 above)
Comment (ncurrier) [reply README.md:16]: we should review that, bowser claims 0.8+ support currently as of last release

Comment (oooookk7) [reply README.md:16]: Ok. I installed it and tested it last month and it didn't work on 10.x, and tested now 10.3.1 and it crashed the moment I opened it. I tested on the office ipad that uses 9.x and it works with getaroom.io.

**Pull request merged**